src/Components/SlateEditor/SlateEditor.js

```diff
@@ -124,6 +166,70 @@ export default function SlateEditor() {
     },
   };
 
+  const getCoordinates = () => {
+    let x, y;
+    const selection = window.getSelection();
+    if (selection.rangeCount !== 0) {
+      const range = selection.getRangeAt(0).cloneRange();
+      range.collapse(false);
+      const rect = range.getClientRects()[0];
+      if (rect) {
+        x = rect.top;
+        y = rect.left;
+      }
+    }
+    return { top: x > 120 ? x - 150 : x + 20, left: y };
```

Comment:
this return value won't make sense if selection.rangeCount == 0 (else case). Fix up the == 0 case also